src/mod_muc_room.erl

```diff
@@ -882,6 +882,11 @@ broadcast_room_packet(From, FromNick, Role, Packet, StateData) ->
             ejabberd_hooks:run(room_send_packet, StateData#state.host, [FilteredPacket, EventData]),
             RouteFrom = jid:replace_resource(StateData#state.jid,
                                              FromNick),
+            RoomName = StateData#state.jid,
+
+            ejabberd_hooks:run(update_inbox, StateData#state.server_host,
```

Comment:
`update_inbox_from_muc`.

btw, usage of map, instead of `[RoomName, {From, RouteFrom}, StateData#state.affiliations, FilteredPacket]` is recommended.